### ISSUE TITLE
Moving login to use keycloak login page

### ISSUE THREAD

**Issue by Vismayak** [IN-CORE/incore-ui#174]
Reason - Enables us to use keycloak APIs to login with 3rd parties
Sources - Farmdoc and Clowder also does this, look for it there
Useful note - https://refactored-carnival-0c1ec87e.pages.github.io/keycloak.html
https://onelinerhub.com/reactjs/how-can-i-implement-keycloak-authentication-with-react-js

**Comment by Vismayak** [comment]
- [ ] Figure out how the passed JWT Token ( Use keycloak.js library)
https://www.npmjs.com/package/keycloak-js

**Comment by Vismayak** [comment]
Update - I have almost figured out how to connect the keycloak js adapter, the issue right now is I initialized it in the wrong place, , right now I created in the actions for reducers. I will instead move the variable to config, then handle the login logic inside the login component similar to farmdoc. Logout handling compnent is created. 

One thing to look out for will be the redirect uri, depending on the place that called the login, this can be handled probably in the component
